templates/resources/worker.md.tmpl

```diff
@@ -0,0 +1,26 @@
+---
+page_title: "boundary_worker Resource - terraform-provider-boundary"
+subcategory: ""
+description: |-
+{{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# boundary_worker (Resource)
+
+{{ .Description | trimspace }}
+
+## Controller-led worker
```

Comment:
Suggestion: What if we organized it to have
```
## Example Usage

### Controller-led worker

### Worker-led worker
```
?

Reply:
Fixed 👌🏼